doc/content/getting_started/contributing.md

```diff
@@ -81,13 +81,13 @@ we will need to create a new branch.
    git checkout devel
    ```
 
-1. Clean and reset devel to get it up to date (make sure you do not have any changes on devel, because you would lose that):
+2. Fetch and reset devel to get it up to date (make sure you do not have any changes on devel, because you would lose that):
```

Comment:
MooseDocs is smart enough to number a list of arbitrary length in the render step, as long as it has a marker that it should number listed items. This feature allows for snippets of this list to be used in an arbitrary fashion elsewhere in the documentation, if needed. "1." serves as that marker, so these should be restored where they were used.